src/LibVLCSharp.Forms/Shared/Themes/Generic.xaml

```diff
@@ -148,6 +148,24 @@
         </Setter>
     </Style>
 
+    <Style x:Key="VideoTracksSelectionButtonStyle" BasedOn="{StaticResource ButtonStyle}" TargetType="Button">
+        <Setter Property="IsVisible" Value="{TemplateBinding IsVideoTracksSelectionButtonVisible}" />
+        <Setter Property="FontFamily" Value="{StaticResource FontAwesomeRegular}" />
+        <Setter Property="Text" Value="{x:Static fontawesome:FontAwesomeIcons.FileVideo}" />
```

Comment:
I'm not sure I'm seeing that button anywhere?

Reply:
Yeah, I will remove it.